src/ray/gcs/gcs_server/test/gcs_table_storage_test.cc

```diff
@@ -0,0 +1,516 @@
+// Copyright 2017 The Ray Authors.
```

Comment:
rename this file to `redis_gcs_table_storage_test`

Reply:
fixed